### DISCUSSION TITLE
How to Change Stoage Location for PhotoPrism LXC

### DISCUSSION THREAD

**Discussion by Luzr** [tteck/Proxmox#2224] [Help Needed] [answered]
Hi,
Where would I change the storage option for PhotoPrism? I have a slower hard drive I want to use for storage and a fast SSD for running PhotoPrism  and my idea is to install PhotoPrism on the SSD.

If it was a Docker container I would [edit ](https://docs.photoprism.app/getting-started/docker-compose/#photoprismstorage)the docker-compose.yml file. With the LXC container I'm unable to find the settings under /bin or /etc (not sure where to look).

Would it be easier to install to the HDD?

Thanks.

**Comment by Luzr** [comment, marked as answer]
Found the settings after searching, not sure how I missed it the first time.
I'll edit it here: /opt/photoprism/config/.env - found the .env file in [this ](https://github.com/tteck/Proxmox/discussions/91)thread. 

Is there away to restart PhotoPrism without rebooting the container?

**Comment by Luzr** [comment]
Is there away to restart PhotoPrism without rebooting the container?

**Comment by geoffpedder** [comment]
for anyone else searching for this, there's a way to do it without re-installing. If you mount your nas to proxmox as storage you can then mount that to the lxc like this:
pct set 101 -mp0 /mnt/pve/truenasScale,mp=/mnt/truenasScaleMount
101 being the number of the lxc
mp0 being the nas mounted on the proxmox host
mp being the mount point you want on the lxc

and then change the paths in the photoprism env /opt/photoprism/config/.env to point to where on your mount point you want to store things
